tools/flaky_test_monitor/common/level3.go

```diff
@@ -0,0 +1,24 @@
+package common
+
+// models full level 3 summary of a test run from 1 (single) level 2 test run
+type TestSummary3 struct {
+	NoResults []TestResultSummary `json:"no_results"`
+
+	// ordered list of tests (from level 2 summary) that:
+	// a) met minimum failure threshold as specified in property file property `failures_threshold_percent`
+	// b) is up to maximum slice size as specified in `failures_slice_max` in property file
```

Comment:
You're saying that if want the "top x failed tests" that would automatically gives us the tests with the highest failure rate, making the min failure threshold redundant?

Reply:
It might be useful to know the total # of tests that have a certain failure threshold but not actually return all of them (e.g. we might have 50 tests with failure rate >= 30% but we want to return just the top 10). This was something I discussed with @Kay-Zee 

It might be useful to know how many tests in total we have above a certain failure rate. Otherwise, we're always returning the "top 10" failed tests but we don't know how many tests we have in total above a certain failure threshold.

Reply:
yeah, I specifically requested this, since I felt like i only wanted details about the top 10 or so of each category, but i also wanted to know if there was something like 50 tests that matched the condition.

we do get two specific pieces of data, (e.g. top 10 worst offenders, and total number of offenders) so i don't think the two are redundant